tf_explain/core/vanilla_gradients.py

```diff
@@ -24,15 +40,66 @@ def explain(self, validation_data, model, class_index):
         Args:
             validation_data (Tuple[np.ndarray, Optional[np.ndarray]]): Validation data
                 to perform the method on. Tuple containing (x, y).
-            model (tf.keras.Model): tf.keras model to inspect
+            model (tf.keras.Model): tf.keras model to inspect. The last two layers of
+                the model should be: a layer which computes class scores with no
+                activation, followed by an activation layer.
```

Comment:
I think it would be nice to explain why we have to do that

Reply:
Ok, I'll add an explanation.